src/public/javascripts/terminal.js

```diff
@@ -193,3 +193,22 @@ function fillConstants() { // eslint-disable-line no-unused-vars
   document.getElementById('hvBonePort').value = consts.hvBone.port;
   document.getElementById('renderInterval').value = consts.renderInterval;
 }
+
+//Torque Value Submission
+
+document.getElementById('torqueSendButton').addEventListener('click', () => { 
+  comms.sendHVCommand('setTorque' + document.getElementById('torqueInput').innerHTML);
+});
+document.getElementById('torqueUpButton').addEventListener('click', incrementTorqueValue);
+document.getElementById('torqueDownButton').addEventListener('click', decrementTorqueValue);
+document.getElementById('terminalEstop').addEventListener('click', () => {
+  comms.sendEBrake();
+});
+
+function incrementTorqueValue() {
+  document.getElementById('torqueInput').innerHTML++;
+}
+
+function decrementTorqueValue() {
```

Comment:
Please add JSDoc Function headers. See [the documentation](https://jsdoc.app/about-block-inline-tags.html) for help